docs/getting-started.md

```diff
@@ -88,6 +89,50 @@ We can use the redis image based on alpine linux from the DockerHub.
 The containerd client uses the `Opts` pattern for many of the method calls.
 We use the `containerd.WithPullUnpack` so that we not only fetch and download the content into containerd's content store but also unpack it into a snapshotter for use as a root filesystem.
 
+Let's put the code together and see it in action.
+
+```go
+package main
+
+import (
+        "context"
+        "log"
+
+        "github.com/containerd/containerd"
+        "github.com/containerd/containerd/namespaces"
+)
+
+func main() {
+        if err := redisExample(); err != nil {
+                log.Fatal(err)
+        }
+}
+
+func redisExample() error {
+        client, err := containerd.New("/run/containerd/containerd.sock")
+        if err != nil {
+                return err
+        }
+        defer client.Close()
+
+        ctx := namespaces.WithNamespace(context.Background(), "example")
+        image, err := client.Pull(ctx, "docker.io/library/redis:alpine", containerd.WithPullUnpack)
+        if err != nil {
+                return err
+        }
+        log.Printf("Successfully pulled %s image\n", image.Name())
+
+        return nil
+}
+```
+
+```bash
+> go build -o example main.go
```

Comment:
The error message indicates that 3 packages are missing. They are "context", "github.com/containerd/containerd", and "github.com/containerd/containerd/namespaces". After a bit of Google searching, the "context" package was added to standard library in go1.7. My guest is you are using go version <1.7. The Developer Quick-Start section at https://github.com/containerd/containerd#developer-quick-start requires go version 1.8.x or above. I suggest to explicitly state that user needs to install go version 1.8.x before following this guide. I think we could append this sentence "You also need Go version 1.8.x or above to follow this guide." at the end of the second paragraph.

The other 2 missing packages can be downloaded using `go get`. But before running `go get`, user needs to set `GOPATH` and the `GOBIN` variables.

Here is step by step guide someone would have to take if they'd not used containerd before and want to get example in this page running.
- Clone this repo
- Copy the sample code to a file such as `main.go`
- Export $GOPATH variable, `export GOPATH=$(go env GOPATH)`
- Export $GOBIN variable, `export GOBIN=$GOPATH/bin`
- Download dependencies, `go get`
- Compile the source, `go build -o example main.go`
- Run the binary, `sudo ./example`